katas/6kyu/detect-pangram.js

```diff
@@ -0,0 +1,5 @@
+function isPangram(string) {
+    const abc = new Set(['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']);
+    const letters = new Set(string.replace(/[^a-z]/gi, '').toLowerCase().split(''));
```

Comment:
it's nice that you think about input validation and make it simple via RegExp. nice job!